components/Section/Section.css

```diff
@@ -0,0 +1,22 @@
+.Section {
+    border: 1px solid black;
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.Dropdown {
+	border: 2px solid black;
+    display: inline;
+    height: 40px;
+    margin: 15px;
+    align-items: center;
```

Comment:
Does the 'align-items' do anything here? I would think display would need to be Flex. Will ask you about it. :)